solution/deps.yaml

```diff
@@ -51,7 +51,7 @@ mongodb:
 pensieve-api:
   sourceRegistry: registry.scality.com/pensieve-api
   image: pensieve-api
-  tag: 1.2.5
+  tag: 1.3.0
   envsubst: PENSIEVE_API_TAG
```

Comment:
should also bump zenko-ui, once [ZKUI-167](https://scality.atlassian.net/browse/ZKUI-167) is merged